<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtVisualLoader1Class</class>
 <widget class="QWidget" name="QtVisualLoader1Class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>747</width>
    <height>725</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VisualLoader</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>C:/Users/Administrator/OneDrive/Desktop/suspacious.png</normaloff>C:/Users/Administrator/OneDrive/Desktop/suspacious.png</iconset>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>511</x>
     <y>330</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="startpro">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>330</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>运行</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>40</y>
     <width>531</width>
     <height>171</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>120</width>
     <height>281</height>
    </rect>
   </property>
   <property name="title">
    <string>GroupBox</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>160</width>
      <height>261</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QRadioButton" name="romax_radioButton">
       <property name="text">
        <string>romax</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="recurdyn_radioButton">
       <property name="text">
        <string>recurdyn</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="adams_radioButton">
       <property name="text">
        <string>adams2018</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="ug_radioButton">
       <property name="text">
        <string>UG 6.0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="AMESim_radioButton">
       <property name="text">
        <string>AMESim</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="sysplore_radioButton">
       <property name="text">
        <string>sysplore</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="ansys_radioButton">
       <property name="text">
        <string>ansys</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QtVisualLoader1.qrc"/>
 </resources>
 <connections/>
</ui>
